app/models/endpoint.rb

```diff
@@ -1,3 +1,9 @@
+# frozen_string_literal: true
+
 class Endpoint < ApplicationRecord
   has_one :account
+
+  def switchable_options
```

Comment:
Can you explain this change more?  Why is this needed?

Reply:
That's the problem with these things being "out" on their own branches so long. I vaguely remember this code.  Let's see what past Rob had to say:

```
commit 825029de758cffaa2425674b88e96eaa91ffbb05
Author: Rob Kaufman <rob@notch8.com>
Date:   Thu Feb 21 12:15:07 2019 -0800

    some endpoints have empty options, clearing (for example) the fedora url, which then causes the process to get and stay stuck
```

I think a code comment to that affect is worth while.

Reply:
we were seeing that if you had a blank solr or fedora url you could basically cause passenger processes to stop responding. that seems like it was worth fixing and trying to future proof a little.

Reply:
:+1: Thanks!